### DISCUSSION TITLE
Adding a new "Desaturated" display mode for the editor.

### DISCUSSION THREAD

**Discussion by Mautar55** [godotengine/godot-proposals#11002] [Editor]
When it comes to materials and lightning in level design, the value (as in HSV color) of the elements of the scene is very relevant, for example, to mark paths in subtle manners. A common way to search for a better configuration is to look at the value of the overall scene and be sure that the relevant items are clearly distinguished from the background and other less relevant items.

My proposal is to add a new display mode for artistic purposes along with the other ones. Currently to do this quickly you have to enable and change the adjustments of the relevant world environment, but this risks changing previously established configurations.

Here are some screenshots to give you an idea:
(ik, not the most impressive example)

Starting scene
![1 starting](https://github.com/user-attachments/assets/9727c344-6729-4317-9ad7-827c1458430c)

New Option
![2 new_option](https://github.com/user-attachments/assets/1eb63942-bd26-4904-9d40-43f59793a3bf)

Preview before adjustment
![3 preview](https://github.com/user-attachments/assets/90a0ffc6-fc19-4827-9b71-52dae3b105c5)

Preview after changing the value of the green material
![4 preview_post](https://github.com/user-attachments/assets/9c811421-e52c-42e7-85e1-cb874c2d0cdb)

End result
![5 end_result](https://github.com/user-attachments/assets/659dba92-9718-4c46-b62e-b5381828321b)

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/473.
